static/coffee/flows/controllers.coffee

```diff
@@ -1836,6 +1847,20 @@ NodeEditorController = ($rootScope, $scope, $modalInstance, $timeout, $log, Flow
   else
     $scope.action._attachType = "image"
 
+  startNodeConfig = () ->
+    $scope.quickReplies = []
+    $scope.action.quick_replies = []
+    $scope.showQuickReplyButton = true
+
+  if $scope.options.dragSource?
+    startNodeConfig()
+  else
+    if $scope.action.quick_replies? and $scope.action.quick_replies != undefined and $scope.action.quick_replies.length > 0
+      $scope.quickReplies = $scope.action.quick_replies
+      $scope.showQuickReplyButton = false
+    else
+      startNodeConfig()
+
```

Comment:
This is really confusing. Maybe just initialize it first and set it if appropriate.

```coffeescript
$scope.quickReplies = []
$scope.action.quick_replies = []
$scope.showQuickReplyButton = true

if $scope.action.quick_replies? and $scope.action.quick_replies.length > 0
    $scope.quickReplies = $scope.action.quick_replies
    $scope.showQuickReplyButton = false

```

Reply:
Doing that the conditional always would be `False` because `$scope.action.quick_replies` is initialized as empty